packages/suite/src/components/wallet/AccountsMenu/AccountItem.tsx

```diff
@@ -106,89 +105,84 @@ interface AccountItemProps {
+export const AccountItem = forwardRef(
+    ({ account, selected, closeMenu }: AccountItemProps, ref: React.Ref<HTMLDivElement>) => {
+        const { goto } = useActions({
+            goto: routerActions.goto,
+        });
+
+        const { accountType, formattedBalance, index, metadata, networkType, symbol, tokens } =
+            account;
+
+        const accountRouteParams = useMemo(
+            () => ({
+                symbol,
+                accountIndex: index,
+                accountType,
+            }),
+            [symbol, index, accountType],
+        );
+
+        const handleClickOnTokens = useCallback<React.MouseEventHandler<HTMLButtonElement>>(
+            event => {
+                event.stopPropagation();
+                closeMenu();
+                goto('wallet-tokens', { params: accountRouteParams });
+            },
+            [accountRouteParams, closeMenu, goto],
+        );
+
+        const dataTestKey = `@account-menu/${symbol}/${accountType}/${index}`;
+
+        return (
+            <Wrapper selected={selected} type={accountType} ref={ref}>
+                <AccountHeader
+                    onClick={() => {
+                        closeMenu();
+                        goto('wallet-index', { params: accountRouteParams });
+                    }}
+                    data-test={dataTestKey}
+                >
+                    <Left>
+                        <CoinLogo size={16} symbol={symbol} />
+                    </Left>
+                    <Right>
+                        <Row>
+                            <AccountName data-test={`${dataTestKey}/label`}>
+                                <AccountLabel
+                                    accountLabel={metadata.accountLabel}
+                                    accountType={accountType}
+                                    symbol={symbol}
+                                    index={index}
                                 />
-                            )}
-                    </Row>
-                    <Row>
-                        <FiatValue
-                            amount={account.formattedBalance}
-                            symbol={account.symbol}
-                            showApproximationIndicator
-                        >
-                            {({ value }) =>
-                                value ? <FiatValueWrapper>{value}</FiatValueWrapper> : null
-                            }
-                        </FiatValue>
-                    </Row>
-                </Right>
-            </AccountHeader>
-        </Wrapper>
-    );
-});
+                            </AccountName>
+                        </Row>
+                        <Row>
+                            <Balance>
+                                <CoinBalance value={formattedBalance} symbol={symbol} />
+                            </Balance>
+                            {(networkType === 'ethereum' || networkType === 'cardano') &&
```

Comment:
It is safer to keep networkType check here, because only those two network types has Tokens tab activated and this has click action navigating there https://github.com/trezor/trezor-suite/blob/feat/coinjoin-default-account-name/packages/suite/src/components/wallet/AccountTopPanel/components/AccountNavigation/index.tsx/#L81

XRP network also has tokens but not yet implemented in Suite.